### ISSUE TITLE
required fields are getting null for `getErrorMessage` instead of the validationError fallback.

### ISSUE THREAD

**Issue by TheMcMurder** [formsy/formsy-react#123]
Required fields are getting null instead of the validationError.

Example:

```jsx
          <Formsy>
            <TestInput name="A"
              ref={el => this.ref = el}
              validationError='Field is required'
              required
            />
          </Formsy>
...
this.ref.getErrorMessage() // returns null instead of `Field is required`
```

PR incoming

**Comment by MilosRasic** [comment]
Amazing that this has gone unnoticed for so long. I guess it's not the most frequently used way of checking for errors. Good catch. Will be reviewing your PR shortly. Sorry for the delay.

**Comment by mchurichi** [comment]
any plan to release this fix soon?

**Comment by rkuykendall** [comment]
This fix was released in Formsy ^2.0.0.
